### ISSUE TITLE
Unfork repo

### ISSUE THREAD

**Issue by dotasek** [cytoscape/cyREST#35]
This repo was originally forked from idekerlab/cyREST, which hasn't been updated since CyREST became a core app. This makes git activity just a little more annoying, so if there's a way to make this independent of idekerlab, that would make like a bit easier.

**Comment by bdemchak** [comment]
Agreed ... let's simply kill the idekerlab version ... this should have been killed when the repo became core. It's overdue.
